stockflux-securities-master/src/components/reusableComponents/TextField.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./TextField.css";
+
+const TextField = ({ value, id, onChange, readOnly }) => {
```

Comment:
It can be worth using the spread operator (`...`) to grab any other props that may be passed through and use the spread operator again to pass them as props to your `<input>`. It might also be worth adding support for passing through a custom `className` when your component is used, which would be appended to your existing `className`

Reply:
okay done this now